pandas/_libs/index.pyx

```diff
@@ -603,35 +603,34 @@ cdef class BaseMultiIndexCodesEngine:
     def _codes_to_ints(self, ndarray[uint64_t] codes) -> np.ndarray:
         raise NotImplementedError("Implemented by subclass")
 
-    def _extract_level_codes(self, ndarray[object] target) -> np.ndarray:
+    def _extract_level_codes(self, target) -> np.ndarray:
```

Comment:
can you type in followon?

Reply:
yah mentioned in OP plans for follow-on to restore common signature/typing